data-services/src/main/java/org/pdxfinder/services/dto/europepmc/Publication.java

```diff
@@ -0,0 +1,28 @@
+package org.pdxfinder.services.dto.europepmc;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Publication {
+
+    private Map<String, List<Result>> resultList;
+
+    public Publication() {
+    }
+
+    public Publication(Map<String, List<Result>> resultList) {
+        this.resultList = resultList;
+    }
+
+    public Map<String, List<Result>> getResultList() {
+        return resultList;
```

Comment:
Initialise resultList as an empty list in the field declaration (ideally) or in constructor otherwise this will cause an NPE if unset